### ISSUE TITLE
a11y high contrast mode issues

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1046]
- [ ] Explore relationships between x and y in z	High Contrast	[A11Y - Health Equity Tracker]: Items on Explore relationships page not demarcated properly in high contrast mode_Web_1.16(d).	"ENVIRONMENT:
Platform: Windows 10 Enterprise
Browser: Firefox Version 88.0.1
Pre Req:
Enable High Contrast Black Mode (Settings > Personalization > Colors > High Contrast Settings)
STEPS TO REPRODUCE:
1). Open https://healthequitytracker.org/exploredata
2). Click on next button to reach the final sub page
3). Verify the options are selected as such, the page is 'Explore relationships between Diabetes and Covid-19 in United States.
4). Observe the items on the page in high contrast mode
EXPECTED RESULT:
Items in the page should be marked properly in High Contrast mode.
OBSERVED RESULT:
1. Selected status of pagination buttons is not marked properly, so user can't know which button is selected at the moment. 
2. Selected status of toggle button under Demographic & Data Type is not marked properly, so user can't know which button is selected at the moment. 
3. Text associated with the color depth index is not visible in high contrast mode.
4. Section borders are not visible in high contrast mode, so a user would never know the page has multiple sections. 
Screenshots attached."	"https://drive.google.com/file/d/1QMKCQUttKtH3GFpOu423DY8-ll_V3zeV/view?usp=sharing
https://drive.google.com/file/d/14fiWXk1u1poBfGt7cxrkfD5wNyvFiTqf/view?usp=sharing
https://drive.google.com/file/d/1j3yipRGj97LuHYCZCFqt_nF1QB7Gd_19/view?usp=sharing
https://drive.google.com/file/d/1VyGMq7NGGJfCiZLbzQd25lIWfhUmJmoZ/view?usp=sharing"	Same issue is observed on Compare rates page.

- [ ] All pages	High Contrast	[A11Y - Health Equity Tracker]: Page up button not demarcated properly in high contrast mode_Web_1.16(d).	"ENVIRONMENT:
Platform: Windows 10 Enterprise
Browser: Firefox Version 88.0.1
Pre Req:
Enable High Contrast Black Mode (Settings > Personalization > Colors > High Contrast Settings)
STEPS TO REPRODUCE:
1). Open https://dev.healthequitytracker.org/datacatalog
2). Observe the page up button under page footer in high contrast mode
EXPECTED RESULT:
Items in the page should be marked properly in High Contrast mode. Page up button should have a border to demarcate it as such and should not be visible as plain text.
OBSERVED RESULT:
Page up button under page footer is demarcated like plain text in high contrast mode which can be confusing for the user.
Screenshot attached."	https://drive.google.com/file/d/1uLelPyvmpiAmDKUjs_bse8rM9nP50nwB/view?usp=sharing	This is applicable across all pages

- [ ] Data Download	High Contrast	[A11Y - Health Equity Tracker]: Items on Data Downloads page not demarcated properly in high contrast mode_Web_1.16(d).	"ENVIRONMENT:
Platform: Windows 10 Enterprise
Browser: Firefox Version 88.0.1
Pre Req:
Enable High Contrast Black Mode (Settings > Personalization > Colors > High Contrast Settings)
STEPS TO REPRODUCE:
1). Open https://dev.healthequitytracker.org/datacatalog
2). Click on Data Downloads tab
3). Observe the items on the page in high contrast mode
EXPECTED RESULT:
Items in the page should be marked properly in High Contrast mode.
OBSERVED RESULT:
1. Selected tab is not marked properly, so user can't know which tab is selected at the moment. 
2. Section borders are not visible in high contrast mode, so a user would never know the page has multiple sections. 
3. Download buttons don't have border and look like plain text in high contrast mode. User wouldn't know action can be performed on the same.
Screenshot attached."	https://drive.google.com/file/d/15SnEIh2PZQ6KJQK-SHevBsX4tGlfg3rw/view?usp=sharing

- [ ] What is Health Equity	High Contrast	[A11Y - Health Equity Tracker]: Items on What is Health Equity page not demarcated properly in high contrast mode_Web_1.16(d).	"ENVIRONMENT:
Platform: Windows 10 Enterprise
Browser: Firefox Version 88.0.1
Pre Req:
Enable High Contrast Black Mode (Settings > Personalization > Colors > High Contrast Settings)
STEPS TO REPRODUCE:
1). Open https://healthequitytracker.org/whatishealthequity
2). Click on What is Health Equity tab
3). Observe the items on the page in high contrast mode
EXPECTED RESULT:
Items in the page should be marked properly in High Contrast mode.
OBSERVED RESULT:
1. Selected tab is not marked properly, so user can't know which tab is selected at the moment. 
2. Expand/ Collapse button don't have border and look like plain text in high contrast mode. User wouldn't know action can be performed on the same.
3. Enter email address edit box doesn't have a border and look like plain text in high contrast mode. User wouldn't know action can be performed on the same.
4. Sign up button doesn't have a border and look like plain text in high contrast mode. User wouldn't know action can be performed on the same.
Screenshots attached."	"https://drive.google.com/file/d/1daaZbBfyXDEwLbwWEdZA178Mz6iuGljT/view?usp=sharing
https://drive.google.com/file/d/1rpDf_7N-uhotGqJTRZMKIC99e9aIzuA7/view?usp=sharing
https://drive.google.com/file/d/1GLdI9kAT7jAgPD1oaPMM3t_2RCFFYauH/view?usp=sharing"	This issue is observed for all related pages.
